hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -84,21 +85,20 @@ def lambda_handler(event, context):
                 MessageBody=action_message.to_aws_message(),
             )
 
-        if threat_exchange_reacting_is_enabled(match_message):
-            threat_exchange_reaction_labels = get_threat_exchange_reaction_labels(
-                match_message, action_labels
-            )
-            if threat_exchange_reaction_labels:
-                for threat_exchange_reaction_label in threat_exchange_reaction_labels:
-                    threat_exchange_reaction_message = (
-                        ReactionMessage.from_match_message_and_label(
-                            match_message, threat_exchange_reaction_label
-                        )
-                    )
-                    config.sqs_client.send_message(
-                        QueueUrl=config.reactions_queue_url,
-                        MessageBody=threat_exchange_reaction_message.to_aws_message(),
+        threat_exchange_reaction_labels = get_reaction_messages(
```

Comment:
threat_exchange_reaction_labels is now a list of reaction messages, so a) this should be renamed and b) I don't understand why you're making another reaction message object at 94. Can't you just call send_message with each reaction message that came back from get_reaction_messages()?